src/Catrobat/Controller/Web/DefaultController.php

```diff
@@ -75,6 +84,15 @@ public function indexAction(Request $request, FeaturedImageRepository $image_rep
       $featured_items = $repository->getFeaturedItems($flavor, 10, 0);
     }
 
+    if ($flavor === 'arduino')
+    {
+      $example_items = $example_repository->getExampleItems('arduino', 10, 0);
+    }
+    else
```

Comment:
only load embroidery projects when the embroidery flavor is used,

might be better to just load examples for the current flavor, if flavors don't have examples just don't show this category. ...->getExampleItems($flavor, 10, 0)